### DISCUSSION TITLE
Admin dashboard

### DISCUSSION THREAD

**Discussion by seoeaa** [serverpod/serverpod#666] [Ideas]
Admin dashboard

Instead of a monitoring module, immediately generate an admin panel

https://github.com/thegera4/web_admin_dashboard_flutter

<a href='https://hostingkartinok.com/show-image.php?id=31046915a0d024ae5b8812f62af0b170' title='поделиться изображение'><img src='https://s1.hostingkartinok.com/uploads/thumbs/2023/02/31046915a0d024ae5b8812f62af0b170.png' alt='Yes' /></a>

**Comment by vlidholt** [comment]
This is something we will be working on for the future. Serverpod Insights is a first step.
